async/deadline_test.ts

```diff
@@ -77,8 +87,13 @@ Deno.test("deadline() handles already aborted signal", async () => {
     .then(() => "Hello");
   const abort = new AbortController();
   abort.abort();
-  await assertRejects(async () => {
-    await deadline(p, 100, { signal: abort.signal });
-  }, DeadlineError);
+  const error = await assertRejects(
+    async () => {
+      await deadline(p, 100, { signal: abort.signal });
+    },
+    DOMException,
+    "The signal has been aborted",
```

Comment:
Hm... The discrepancy between the error message in this test and the one above seems odd.